app/page/template/list/preferences.htm

```diff
@@ -5,8 +5,12 @@
   </div>
   <div class="left-list-center">
     <ul class="left-list-items preferences-list-items">
-      <li class="left-list-item left-list-item-clickable" data-bind="click: click_on_account, css: {'text-theme': selected_account}" data-uie-name="go-account">
-        <div class="left-column icon-account"></div>
+      <li class="left-list-item left-list-item-clickable" data-bind="click: click_on_account, css: {'text-theme': selected_account, 'svg-theme': selected_account}" data-uie-name="go-account">
+        <div class="left-column">
+            <svg width="16" height="16" viewBox="0 0 16 16">
+              <path class="fill-theme" d="M8 16A8 8 0 1 1 8 0a8 8 0 0 1 0 16zM2 8c0 1.5.56 2.88 1.47 3.94l.08-.46c.15-.82.93-1.48 1.76-1.48h5.38c.83 0 1.61.67 1.76 1.48l.08.46A6 6 0 1 0 2 8zm6 1a2.5 2.5 0 1 1 0-5 2.5 2.5 0 0 1 0 5z"/>
```

Comment:
Edge doesn't like self closing paths and throws errors in the console. Please adjust for that.